### DISCUSSION TITLE
Option to cover full space when just a single window in workspace

### DISCUSSION THREAD

**Discussion by zefr0x** [YaLTeR/niri#787] [Ideas]
Somewhat like `always-center-single-column`, when we just have a single window in a workspace, I would like to have it maximized like `maximize-column`, and its `gaps` are set to `0`.

Also the window should have `geometry-corner-radius` set to `0` so we don't have portion of the focus-ring with an empty space in every corner.

Just like the `no_gaps_when_only` option in Hyprland.

**Comment by zefr0x** [comment]
Sorry, I searched for duplicates in the issues but not the discussions.

**Reply by YaLTeR** [reply]
Could you link the discussion here for anyone searching?

**Reply by zefr0x** [reply]
#508
